docs/source/notes/quick_start.rst

```diff
@@ -1,10 +1,9 @@
 Quickstart and introduction by example
 ======================================
 
-As discussed in the :ref:`whatisal` section, the **PyRelationAL** package decomposes the active learning workflow into four
-main components: 1) a data manager, 2) a model, 3) an acquisition strategy built around an informativeness measure,
-and 4) an oracle. Note however, that the oracle is external to the package. In this section, we work through an example
-to illustrate how to instantiate and combine a data manager, a model, and an acquisition strategy.
+As discussed in the :ref:`whatisal` section, the **PyRelationAL** package decomposes the active learning workflow into five
+main components: 1) a data manager, 2) a model manager, 3) an acquisition strategy built around an informativeness measure, 4) an oracle and 5) a pipeline.
+In this section, we work through an example to illustrate how to instantiate and combine a data manager, a model manager, an acquisition strategy and an oracle.
 
 Data Manager
 ------------
```

Comment:
we should add a mention here that the pyrelational datamanager makes some assumptions about dataset. e.g

- it assumes that the labels are contained in an independent attribute of the dataset (unlike TensorDatasets) that can be specified with the `label_attr` input to the data manager
- that the dataset __getitem __ return a tuple of tensors
- a number of methods assumes the featurization is entirely contained in a single tensor, corresponding to the first element of the tuple returned by the __getitem __

Reply:
maybe to add to the readme too

Reply:
good points, I have added it here, in the doc string and in the readme